x-pack/dockerlogbeat/pipelinemanager/clientLogReader.go

```diff
@@ -103,7 +103,7 @@ func (cl *ClientLogger) publishLoop(reader chan logdriver.LogEntry) {
 				"container": common.MapStr{
 					"labels": cl.containerMeta.ContainerLabels,
 					"id":     cl.containerMeta.ContainerID,
-					"name":   cl.containerMeta.ContainerName,
+					"name":   strings.Trim(cl.containerMeta.ContainerName, "/"),
```

Comment:
Would it be safer to cover it with a simple test case?

Reply:
You mean support this with a unit test?

Reply:
Yes, thanks.